pkg/config/config.go

```diff
@@ -389,6 +389,8 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("logs_config.run_path", defaultRunPath)
 	config.BindEnv("logs_config.dd_url")
 	config.BindEnvAndSetDefault("logs_config.use_http", false)
+	config.BindEnvAndSetDefault("logs_config.use_compression", false)
+	config.BindEnvAndSetDefault("logs_config.compression_level", 6) // Default level for the gzip/deflate algorithm
```

Comment:
Could we also add config parameters for linger time?
It's great to use compression_level to let users pick how much cpu they want to trade.
I'd also expose how much additional latency they want to add at most.

Reply:
then we probably want to make the batch size configurable too ?

Reply:
In my opinion it is a bit orthogonal with this PR but it is worth exploring that.

Just to make things clearer, here we are implementing compression for Hulu with a deadline at the end of the month so we are tacking some shortcuts. But after we should do proper benchmark of compression algorithm, batch size, linger time, and so on.

Reply:
I think the batch size is a little different, it depends a lot on our intake so users don't care much.
But linger time sounds important as it means controlling how much additional latency we add. Sounds worth adding in a first version to me. What's the current (default) value?

Reply:
The current linger time is 5s.
I will add the configuration of the linger time in another PR with a [1s:10s] authorised range.
Sound good to you @tmichelet?

Reply:
ok